### DISCUSSION TITLE
Welcome to the Dashlane CLI discussions!

### DISCUSSION THREAD

**Discussion by Mikescops** [Dashlane/dashlane-cli#123] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Get help on your projects using the Dashlane CLI.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

If you need support related to other Dashlane apps or your own account, please go to our [Dashlane Help Center](https://support.dashlane.com/hc/en-us)!

**Comment by BenjaminOddou** [comment]
Hey 👋, thanks @Mikescops for the post, it's a great idea. To present myself rapidly, I am the author of [Alfred-Dashlane](https://github.com/BenjaminOddou/alfred-dashlane) that is built on top of the cli. This workflow is working in [Alfred](https://www.alfredapp.com), a productivity app for Mac users.

I have some ideas that I'd like to share:
- I hope to see Dashlane-cli officially integrated into Homebrew core one day in order to distribute it more easily.
- Having the possibility to modify add or even delete a password / otp / note could be a good evolution !
- Having the possibility to add multiple vaults

Happy to discuss with you ! 😉

**Reply by Mikescops** [reply]
Thanks for sharing! Nice work, I'll give it a try 😃 

- We now have a dedicated homebrew tap which makes it much easier to get updates on your side : `brew install dashlane/tap/dashlane-cli`
- Write operations on vault are definitely the way to go, tho' it requires much more work and maintenance, we'll get there 
- Yes, I'm looking for that too, you can open an issue for it 👍

**Reply by BenjaminOddou** [reply]
Thanks for the response @Mikescops ! Don't hesitate to share your feedbacks and expertise regarding the Alfred project.

Concerning the tap, it's great but as mentioned [here](https://www.alfredforum.com/topic/19994-dashlane/#comment-107379) it requires Xcode to build the binary which doesn't help me to distribute the workflow.

It's just something I wish for the future ! But it's already nice to have this solution.

**Reply by Mikescops** [reply]
Ah yes good catch, I'll try to figure out how to skip this requirement, so far it's mandatory as it used to self sign the final package so your kernel does not reject the binary.

**Reply by BenjaminOddou** [reply]
The --build-from-source flag is still working but not ideal. That's the solution I chose in my project.

- [install.sh](https://github.com/BenjaminOddou/alfred-dashlane/blob/main/install.sh)
- [dashlane-cli.rb](https://github.com/BenjaminOddou/alfred-dashlane/blob/main/src/dashlane-cli.rb)
